x-pack/plugins/profiling/public/components/no_data_page.tsx

```diff
@@ -124,7 +124,8 @@ docker.elastic.co/observability/profiling-agent:stable /root/pf-host-agent \\
           }),
           content: (
             <EuiCodeBlock paddingSize="s" isCopyable>
-              wget -O- https://releases.prodfiler.com/stable/pf-host-agent_linux_amd64.tgz | tar xz
+              wget -O pf-host-agent.tgz &quot;https://ela.st/pf-host-agent-amd64&quot; && tar xzf
```

Comment:
Can you explain why you changed this ?

Reply:
It's been requested in https://github.com/elastic/prodfiler/issues/2961#issue-1574328032

Reply:
Hm, it has been requested without giving a reason. Would be interesting to know. Maybe some OSX or zsh incompatibility ? Do you remember or can you find out ?

Reply:
Maybe there's a misunderstanding here 😄 
The major change here is the use of a different URL, a redirect.
The redirect URL _does not have a file extension_, so the downloaded file piped into `tar` would fail.
Aside from the URL, we mainly changed the wget option to download to a file instead of STDOUT.

Reply:
When you pipe the wget output to tar, you don't need any file extension.
Just try it out yourself:
```
wget -O- https://ela.st/pf-host-agent-amd64 | tar xz
```
works like a charm without having to store an intermediate file.